src/Chainweb/Version.hs

```diff
@@ -913,6 +914,14 @@ doCheckTxHash Development = (>= 110)
 doCheckTxHash (FastTimedCPM g) | g == petersonChainGraph = (>= 7)
 doCheckTxHash _ = const False
 
+-- | Placeholder name for 2.13 Pact changes
+chainweb213Pact :: ChainwebVersion -> BlockHeight -> Bool
+chainweb213Pact Mainnet01 = const False -- TODO
+chainweb213Pact Testnet04 = const False -- TODO
```

Comment:
This is what is used in #1358

```Haskell
pactForceEof Mainnet01 = (>= 2_447_358) -- 2022-02-26 00:00:18
pactForceEof Testnet04 = (>= 1_977_816) -- 2022-02-25 00:00:18
```

Reply:
To clarify -- I'm proposing `chainweb213Pact` to be the only fork and used for force-eof, module gas etc.